VoxelisedSimulation/SubScripts/README.md

```diff
@@ -20,7 +20,8 @@ This directory contains a collection of scripts that are key to automatically fi
 
 Scripts
 =======
-* `EstimateGATESTIRNorm.sh`: Creates normalisation correction sinogram from GATE measured data and STIR model data. 
+* `EstimateGATESTIRNorm.sh`: Creates normalisation correction sinogram from GATE measured data and STIR model data.<br />
+   *Warning:* You will need to make sure that the projector that you use during reconstruction matches the one used for the estimation of the norm factors. See `forward_projector_proj_matrix_ray_tracing.par`.
```

Comment:
Does `forward_projector_proj_matrix_ray_tracing.par` exist?

Reply:
it does now!